src/scripts/dfnshelper.js

```diff
@@ -1,99 +1,56 @@
 import globalize from 'globalize';
 
+const dateLocales = (locale) => ({
+    'ar': ar,
+    'be-by': be,
+    'bg-bg': bg,
+    'ca': ca,
+    'cs': cs,
+    'da': da,
+    'de': de,
+    'el': el,
+    'en-gb': enGB,
+    'en-us': enUS,
+    'es': es,
+    'es-ar': es,
+    'es-mx': es,
+    'fa': faIR,
+    'fi': fi,
+    'fr': fr,
+    'fr-ca': frCA,
+    'gsw': de,
+    'he': he,
+    'hi-in': hi,
+    'hr': hr,
+    'hu': hu,
+    'id': id,
+    'it': it,
+    'kk': kk,
+    'ko': ko,
+    'lt-lt': lt,
+    'ms': ms,
+    'nb': nb,
+    'nl': nl,
+    'pl': pl,
+    'pt-br': ptBR,
+    'pt-pt': pt,
+    'ro': ro,
+    'ru': ru,
+    'sk': sk,
+    'sl-si': sl,
+    'sv': sv,
+    'tr': tr,
+    'uk': uk,
+    'vi': vi,
+    'zh-cn': zhCN,
+    'zh-hk': zhCN,
+    'zh-tw': zhTW
+})[locale];
+
 export function getLocale() {
-    switch (globalize.getCurrentLocale()) {
-        case 'ar':
-            return ar;
-        case 'be-by':
-            return be;
-        case 'bg-bg':
-            return bg;
-        case 'ca':
-            return ca;
-        case 'cs':
-            return cs;
-        case 'da':
-            return da;
-        case 'de':
-            return de;
-        case 'el':
-            return el;
-        case 'en-gb':
-            return enGB;
-        case 'en-us':
-            return enUS;
-        case 'es':
-            return es;
-        case 'es-ar':
-            return es;
-        case 'es-mx':
-            return es;
-        case 'fa':
-            return faIR;
-        case 'fi':
-            return fi;
-        case 'fr':
-            return fr;
-        case 'fr-ca':
-            return frCA;
-        case 'gsw':
-            return de;
-        case 'he':
-            return he;
-        case 'hi-in':
-            return hi;
-        case 'hr':
-            return hr;
-        case 'hu':
-            return hu;
-        case 'id':
-            return id;
-        case 'it':
-            return it;
-        case 'kk':
-            return kk;
-        case 'ko':
-            return ko;
-        case 'lt-lt':
-            return lt;
-        case 'ms':
-            return ms;
-        case 'nb':
-            return nb;
-        case 'nl':
-            return nl;
-        case 'pl':
-            return pl;
-        case 'pt-br':
-            return ptBR;
-        case 'pt-pt':
-            return pt;
-        case 'ro':
-            return ro;
-        case 'ru':
-            return ru;
-        case 'sk':
-            return sk;
-        case 'sl-si':
-            return sl;
-        case 'sv':
-            return sv;
-        case 'tr':
-            return tr;
-        case 'uk':
-            return uk;
-        case 'vi':
-            return vi;
-        case 'zh-cn':
-            return zhCN;
-        case 'zh-hk':
-            return zhCN;
-        case 'zh-tw':
-            return zhTW;
-        default:
-            return enUS;
-    }
+    return dateLocales(globalize.getCurrentLocale() || enUS);

```

Comment:
_visually_
```suggestion
    return dateLocales(globalize.getCurrentLocale()) || enUS;
```

Reply:
Btw, does `globalize.getCurrentDateTimeLocale` make sense now?

Reply:
All the dates aren't converted yet (in fact, the only thing using date-fns currently is the activity log) so it still has uses :)